contracts/StablecoinConverter.sol

```diff
@@ -13,29 +13,29 @@ import "./libraries/TokenConservation.sol";
  *  @author @gnosis/dfusion-team <https://github.com/orgs/gnosis/teams/dfusion-team/members>
  */
 contract StablecoinConverter is EpochTokenLocker {
-    using SafeCast for uint;
+    using SafeCast for uint256;
     using SafeMath for uint128;
     using BytesLib for bytes32;
     using BytesLib for bytes;
     using TokenConservation for int[];
     using TokenConservation for uint16[];
     using IterableAppendOnlySet for IterableAppendOnlySet.Data;
 
-    uint constant private MAX_UINT128 = 2**128 - 1;
+    uint128 constant private MAX_UINT128 = 2**128 - 1;
```

Comment:
Is it just me or is this constant unused and can be removed?

Reply:
This might be unused and lets please remove it here!